tensorrt_llm/_torch/auto_deploy/llm_args.py

```diff
@@ -165,8 +165,15 @@ class AutoDeployConfig(DynamicYamlMixInForSettings, BaseSettings):
         "AutoDeployConfig.",
     )
 
+    support_partial_config: bool = Field(
```

Comment:
We are in the process of deprecating fields in llm_args.py that are transform specific. Please instead use the transform-specific config classes and put those config values/fields in default.yaml